keyboards/ztboards/after/info.json

```diff
@@ -1,12 +1,12 @@
 {
```

Comment:
The ordering seems to be wrong here. You can verify it in Configurator by pressing Ctrl+Shift+I to preview the info.json, and turning on Fast Input.
```json
{
    "keyboard_name": "ZTBoards After",
    "url": "ztb.mx",
    "maintainer": "ZTBoards phlop",
    "width": 16.5,
    "height": 5.75,
    "layouts": {
        "LAYOUT_all": {
            "layout": [
                {"x": 0, "y": 0.5},
                {"x": 1, "y": 0.5},
                {"x": 2, "y": 0.5},
                {"x": 3, "y": 0.5},
                {"x": 4, "y": 0.5},
                {"x": 5, "y": 0.5},
                {"x": 6, "y": 0.5},
                {"x": 7, "y": 0.5},
                {"x": 8, "y": 0.5},
                {"x": 9, "y": 0.5},
                {"x": 10, "y": 0.5},
                {"x": 11, "y": 0.5},
                {"x": 12, "y": 0.5},
                {"x": 13, "y": 0.5},
                {"x": 14, "y": 0.5},
                {"x": 15.5, "y": 0},

                {"x": 0, "y": 1.5, "w": 1.5},
                {"x": 1.5, "y": 1.5},
                {"x": 2.5, "y": 1.5},
                {"x": 3.5, "y": 1.5},
                {"x": 4.5, "y": 1.5},
                {"x": 5.5, "y": 1.5},
                {"x": 6.5, "y": 1.5},
                {"x": 7.5, "y": 1.5},
                {"x": 8.5, "y": 1.5},
                {"x": 9.5, "y": 1.5},
                {"x": 10.5, "y": 1.5},
                {"x": 11.5, "y": 1.5},
                {"x": 12.5, "y": 1.5},
                {"x": 13.5, "y": 1.5, "w": 1.5},

                {"x": 15.5, "y": 1.5},

                {"x": 0, "y": 2.5, "w": 1.75},
                {"x": 1.75, "y": 2.5},
                {"x": 2.75, "y": 2.5},
                {"x": 3.75, "y": 2.5},
                {"x": 4.75, "y": 2.5},
                {"x": 5.75, "y": 2.5},
                {"x": 6.75, "y": 2.5},
                {"x": 7.75, "y": 2.5},
                {"x": 8.75, "y": 2.5},
                {"x": 9.75, "y": 2.5},
                {"x": 10.75, "y": 2.5},
                {"x": 11.75, "y": 2.5},
                {"x": 12.75, "y": 2.5, "w": 2.25},

                {"x": 15.5, "y": 2.5},

                {"x": 0, "y": 3.5, "w": 2.25},
                {"x": 2.25, "y": 3.5},
                {"x": 3.25, "y": 3.5},
                {"x": 4.25, "y": 3.5},
                {"x": 5.25, "y": 3.5},
                {"x": 6.25, "y": 3.5},
                {"x": 7.25, "y": 3.5},
                {"x": 8.25, "y": 3.5},
                {"x": 9.25, "y": 3.5},
                {"x": 10.25, "y": 3.5},
                {"x": 11.25, "y": 3.5},
                {"x": 12.25, "y": 3.5, "w": 1.75},

                {"x": 14.25, "y": 3.75},

                {"x": 15.5, "y": 3.5},

                {"x": 0, "y": 4.5, "w": 1.5},

                {"x": 2.25, "y": 4.5, "w": 1.5},
                {"x": 3.75, "y": 4.5, "w": 6.25},
                {"x": 10, "y": 4.5, "w": 1.5},
                {"x": 11.5, "y": 4.5, "w": 1.5},

                {"x": 13.25, "y": 4.75},
                {"x": 14.25, "y": 4.75},
                {"x": 15.25, "y": 4.75}
            ]
        }
    }
}
```

Reply:
Sorry, could you clarify what you mean by "the ordering seems to be wrong"?

Reply:
When converting KLE to the QMK info.json format, the ordering for keys not on the 1u vertical grid will be incorrect, as they are considered by KLE to be on their own row. In this case the rotary encoder key and the up arrow are one row off:
![image](https://user-images.githubusercontent.com/4781841/98884834-fb2b6280-24e4-11eb-855d-45d68e0c49f1.png)
If you turn on Fast Input in the Configurator and spam a key, you can check that the ordering is correct vs. the top half of the LAYOUT macro.

Reply:
Hello, please check now, it should be in the correct order.